### ISSUE TITLE
Errors out when run because PyPDF has deprecated some classes

### ISSUE THREAD

**Issue by talwrii** [pdf-utilities/inject_javascript#9]
**Describe the bug**

Does not work at all because it complaints about deprecated pypdf classes.

**To Reproduce**

* Create a new virtualenv
* Install
* Try to run it

**Expected behavior**

It to run.

**Additional context**

Note that PyPDF2 is now pypdf.

**Comment by talwrii** [comment]
#8 is a PR for this.

**Comment by S0AndS0** [comment]
According to the [change-log][1] version `1.28.0` is when they renamed things;

> - Rename of core classes:
>   - PdfFileReader ➔ PdfReader
>   - PdfFileWriter ➔ PdfWriter
>   - PdfFileMerger ➔ PdfMerger

... And scrolling down further shows version `1.27.12` likely be the last one with expected names.  But it seems there may have been additional changes within Python, specifically how `with open` behaves, because I'm popping errors similar to the following when running tests within a virtual environment with pinned dependency version;

```
(0.0.9) [$USER@$HOSTNAME inject_javascript]$ inject-pdf-javascript --pdf-path ~/Downloads/blank.pdf --js-path dev/hello-world.js --save-path dev/hello-world.pdf --verbose
Notice: 60 return_pdf_data
        finished loading data into output object from: ~/Downloads/blank.pdf
Notice: 81 return_js_data
        finished loading JavaScript data from: dev/hello-world.js
Traceback (most recent call last):
  File "~/git/hub/pdf-utilities/inject_javascript/venv/0.0.9/lib/python3.10/site-packages/inject_javascript/__init__.py", line 93, in save_combined_data
    pdf_data.write(tmp)
# ...
ValueError: I/O operation on closed file: ~/Downloads/blank.pdf
```

And while I managed to find a [StackOverflow Q&A][2] that seemed relevant, adopting the suggested changes would require a significant re-write of class logic :upside_down_face: 

**TLDR** I haven't forgotten about ya, and I'll try to find time again to resolve underlying issues in the future!

[1]: https://pypdf2.readthedocs.io/en/latest/meta/changelog-v1.html#version-1-28-0-2022-05-22
[2]: https://stackoverflow.com/questions/6773631/problem-with-closing-python-pypdf-writing-getting-a-valueerror-i-o-operation
